Classes/UT3DmgType_HellbenderRearGunBeam.uc

```diff
@@ -0,0 +1,11 @@
+class UT3DmgType_HellbenderRearGunBeam extends VehicleDamageType
+	abstract;
+
+defaultproperties
+{
+     VehicleClass=Class'UT3Vehicles.UT3HellbenderRearGunPawn'
+     DeathString="%o was incinerated by %k's twin beams."

```

Comment:
This is not quite the message you get in UT3. Check what it is there or look at the contents of UT3 `.int` files, searching for `KILLS_HELLBENDERPRIMARY`, `DEATHS_HELLBENDERPRIMARY` and `SUICIDES_HELLBENDERPRIMARY` (see http://greatemerald.xmpcommunity.com/uncodex/UT3-15/Source_utgamecontent/utdmgtype_hellbenderprimary.html).

Reply:
I have no clue where to find them, I've tried looking on your Uncode page but can't find anything, and where would the int files be?  (in-game console doesn't work either as I'm not seeing any death messages on weapon/vehicle kills, just team switch, orb carrier, and other specials)